src/web-auth/captcha.js

```diff
@@ -306,8 +303,13 @@ function handleCaptchaProvider(element, options, challenge) {
   );
 }
 
-function runArkose() {
+function triggerCaptcha(callback) {
```

Comment:
Should we add a form of guard clause here? If the caller forgets to check `requiresTrigger()` before invoking this method... Something like:
```suggestion
function triggerCaptcha(callback) {
  if (!requiresTrigger()) {
    return callback();
  }
```

Reply:
Caller doesn't have to check requiresTrigger(). It's more for us but I can see maybe called accidentally calling this without actually using Arkose as provider so that would cause errors. I'll add the guard.